extensions/openai/models.py

```diff
@@ -57,6 +58,8 @@ def _load_model(data):
 
     shared.model, shared.tokenizer = load_model(model_name)
     shared.model_name = model_name
+    if shared.args.lora:
+        add_lora_to_model(shared.args.lora)
```

Comment:
I'm thinking that it would be better to have a new endpoint for applying LoRAs. It would take a list of lora names as input and call the "apply_lora" function in modules/LoRA.py.